src/Razor/src/Microsoft.VisualStudio.RazorExtension/UnifiedSettings/razor.registration.json

```diff
@@ -0,0 +1,102 @@
+{
+  "properties": {
+    "textEditor.razor.advanced.formatOnType": {
+      "type": "boolean",
+      "default": true,
+      "title": "%Setting_FormattingOnTypeDisplayName%",
+      "description": "%Setting_FormattingOnTypeDescription%",
+      "migration": {
+        "input": {
+          "store": "VsUserSettingsRegistry",
+          "path": "Razor\\FormatOnType",
+          "bitmask": 1
```

Comment:
Specifying a bitmask should only be done when you have multiple settings sharing a single legacy store value (where each setting corresponds to a subset of the bits in that value).